style.css

```diff
@@ -1 +1,285 @@
-/*  Add your styling here */
+:root {
+
+}
+
+* {
+    box-sizing: border-box;
+}
+
+body {
+    margin: 0;
+    font-family: 'Montserrat', sans-serif;
+}
+
+nav {
+    display: none;
+}
+
+/* HEADER   */
+
+.header-top-section {
+    background-color: rgb(247, 247, 247);
+    display: flex;
+    justify-content: space-between;
+    height: 150px;
+    padding: 20px 20px 10px 20px;
+    gap: 100px;
+}
+
+.header-align {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-end;
+    align-content: space-between;
+    justify-content: space-between;
+}
+
+.logo {
+    width: 80px;
+    height: 80px;
+}
+
+h1{ 
+    font-size: 0.75em;
+    font-weight: 300;
+    text-align: right;
+    /* padding: 0 30px 30px 0; */
+}
+
+/* MAIN */
+
+main {
+    margin: 20px;
+    /* overflow: hidden; */
+}
+
+/* WELCOME */
+
+.welcome-section,.welcome-text {
+    margin-bottom: 30px;
+}
+
+.welcome-img {
+    width: 99.9%;
```

Comment:
Why was this necessary? 100% not working? 😄

Reply:
lol if I remember correctly, 100% for some reason wasn't working/doing what I needed. I may just double check it again because I rejigged the code again so it may fix the issue and make 100% work.